docs/compute_resources.md

```diff
@@ -28,8 +28,6 @@ Refer to the [EKS Managed Node Group documentation](https://docs.aws.amazon.com/
 ```hcl
   eks_managed_node_groups = {
     bottlerocket_default = {
-      use_custom_launch_template = false
```

Comment:
Line deleted as I feel it is a bit misleading and unrelated to how to use Bottlerocket.

Reply:
this is intended - this is showing using the bottlerocket AMI with the default EKS managed node group launch template

Reply:
revert this change and we can merge the doc changes proposed

Reply:
Done!